packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -75,11 +75,11 @@ export type ClickableHandlers = {|
     onKeyUp: (e: SyntheticKeyboardEvent<*>) => void,
     onFocus: (e: SyntheticFocusEvent<*>) => void,
     onBlur: (e: SyntheticFocusEvent<*>) => void,
-    tabIndex: number,
+    tabIndex?: number,
 |};
 
 const disabledHandlers = {
-    onClick: () => void 0,
+    onClick: (e) => e.preventDefault(),
```

Comment:
This prevents navigation on links.  The other option is to remove the `href`.   How do people feel about this change?

Reply:
I think we should be removing the `href` because that's how browsers know not to navigate. This preventDefault feels like hidden behavior. I think adding an `onClick` to disabled items also made them focusable by mouse press -- the disabled menu items now receive a focus ring.

Reply:
That's a good point about it being kind of hidden.  If you inspected the DOM the link would look like it should work but it wouldn't work and that would be confusing.

Reply:
See https://github.com/Khan/wonder-blocks/pull/251.